src/main/scala/eventstore/PersistentSubscriptionActor.scala

```diff
@@ -0,0 +1,211 @@
+package eventstore
+import akka.actor.Status.Failure
+import akka.actor.{ ActorRef, Props }
+import eventstore.PersistentSubscription.{ Ack, EventAppeared }
+import eventstore.ReadDirection.Forward
+import eventstore.{ PersistentSubscription => PS }
+
+import scala.annotation.tailrec
+import scala.collection.immutable.Queue
+
+object PersistentSubscriptionActor {
```

Comment:
please test use case when physical connection temporary unavailable to EventStore,
`PersistentSubscriptionActor` should handle and that recover successfully